### PR TITLE
Add mimetype wildcard check

### LINKED ISSUE #959: MediaTypes check add wildcard support for image/*, audio/*, video/*
currently only audio/mpeg and video/mp4 are set correctly as video/audio media type.
all mimetypes with audio/* and video/* should be set correctly.

### PR DESCRIPTION
**Pull request by alexander-schranz**
__Tasks:__

- [x] test coverage
- [ ] gather feedback for my changes
- [x] make media types configurable
- [x] add update media type command
- [x] submit changes to the documentation
- [x] added changelog line

__Informations:__

| Q             | A
| ------------- | ---
| Tests pass?   | 
| Fixed tickets | fixes #959
| BC Breaks     | yes (old files can have false media type)
| Doc           | https://github.com/sulu-io/docs/blob/master/detail-specification/500-assets/DET-508-MediaTypes.md

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/Media/TypeManager/DefaultTypeManager.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+namespace Sulu\Bundle\MediaBundle\Media\TypeManager;
+
+use Doctrine\ORM\EntityManager;
+use Sulu\Bundle\MediaBundle\Entity\MediaType;
+use Sulu\Bundle\MediaBundle\Media\Exception\MediaTypeNotFoundException;
+
+class DefaultTypeManager implements TypeManagerInterface
+{
+
+    /**
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Media/TypeManager/DefaultTypeManager.php]: remove empty line

File: src/Sulu/Bundle/MediaBundle/Media/TypeManager/DefaultTypeManager.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+     */
+    private $mediaTypeEntities;
+
+    /**
+     * @param EntityManager $em
+     * @param array $mediaTypes
+     * @param array $blockedMimeTypes
+     */
+    public function __construct(
+        EntityManager $em,
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Media/TypeManager/DefaultTypeManager.php]: do you really need an `EntityManager` or is `ObjectManager` enough?

File: src/Sulu/Bundle/MediaBundle/Media/TypeManager/DefaultTypeManager.php [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        $type = $this->em->getRepository(self::ENTITY_NAME_MEDIATYPE)->find($id);
+        if (!$type) {
+            throw new MediaTypeNotFoundException('Collection Type with the ID ' . $id . ' not found');
+        }
+
+        return $type;
+    }
+
+    /**
+     * @param string $fileMimeType
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Media/TypeManager/DefaultTypeManager.php]: please write here for function phpdoc:
```
/**
 * {@inheritdoc}
 */
```

File: src/Sulu/Bundle/MediaBundle/Media/TypeManager/TypeManagerInterface.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+namespace Sulu\Bundle\MediaBundle\Media\TypeManager;
+
+use Sulu\Bundle\MediaBundle\Entity\MediaType;
+
+interface TypeManagerInterface
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Media/TypeManager/TypeManagerInterface.php]: phpdoc

File: src/Sulu/Bundle/MediaBundle/Media/TypeManager/TypeManagerInterface.php [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+{
+    const ENTITY_CLASS_MEDIATYPE = 'Sulu\Bundle\MediaBundle\Entity\MediaType';
+    const ENTITY_NAME_MEDIATYPE = 'SuluMediaBundle:MediaType';
+
+    /**
+     * @param int $id
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Media/TypeManager/TypeManagerInterface.php]: description please(=

File: src/Sulu/Bundle/MediaBundle/Media/TypeManager/TypeManagerInterface.php (hunk 4 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Media/TypeManager/TypeManagerInterface.php]: can we provide tests for this class?

File: src/Sulu/Bundle/MediaBundle/Media/TypeManager/DefaultTypeManager.php [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\TypeManager;
+
+use Doctrine\ORM\EntityManager;
+use Sulu\Bundle\MediaBundle\Entity\MediaType;
+use Sulu\Bundle\MediaBundle\Media\Exception\MediaTypeNotFoundException;
+
+class DefaultTypeManager implements TypeManagerInterface
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Media/TypeManager/DefaultTypeManager.php]: phpdoc

**Comment by alexander-schranz** [comment]
@wachterjohannes  code updated and test added

**Comment by wachterjohannes** [comment]
ACK if there is no dependency!

**Pull request merged**